### DISCUSSION TITLE
轻量级参数解析库-tiny_cmdline

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#319] [Announcements]
# 轻量级参数解析库-tiny_cmdline

当编写一些命令行版本软件的时候，往往需要涉及到命令行参数的处理。网上能搜索到一些参数处理库，但是对很多需求来说，它们太过庞大，一般也只适合作为黑盒使用。
因此，我编写了一个轻量级的参数解析库，tiny_cmdline，目的就是轻量，让用户容易阅读和定制。

https://imcbc.cn/202408/about_tiny_summary/

**Comment by DeguiLiu** [comment]
很好。
看到一些开源库直接引用的是命令行参数解析库TCLAP，因为你不知道什么时候代码就要求跨平台了。重新修改免不了重新测试。
参考https://blog.csdn.net/stallion5632/article/details/144291087
